bloom-housing/bloom
# refactor: organize design system text styles

# Pull Request Template

## Issue Overview

This PR addresses #2408


## Description

* Renamed for clarity and organized all of our text styles in `text.scss`
* Created a new top level section called `Utilities` that holds the docs for these Typography variants
* Moved the `Heading` component to the `Text` section of Storybook and picked up the new class names in it
* Lots of single class name changes just to handle the renaming
* Created new Bloom theming variables for text colors
* Expanded the already existing custom variables in each text style

## How Can This Be Tested/Reviewed?

Check out the new [docs](https://deploy-preview-3055--dev-storybook-bloom.netlify.app/?path=/story/theming-typography--page)! SF currently is having to re-create some of our text styles, so this will provide a clearer, cleaner, and documented way for them to just pull in our class names.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
